config/locales/de.yml

```diff
@@ -514,6 +514,7 @@ de:
   omniauth:
     google: Google
     access_denied: 'Zutritt verweigert'
+    facebook: Facebook
```

Comment:
We don't need to add this since it already exists: https://github.com/ifmeorg/ifme/blob/master/config/locales/en.yml#L46